docs/Developers/Writing_Native_Bears.rst

```diff
@@ -261,6 +261,9 @@ The Setting does support some very basic types:
 -  List of strings (``list``, values will be split by comma)
 -  Dict of strings (``dict``, values will be split by comma and colon)
 
+You can use shortcuts for basic types, ``str_list`` for String,
+``int_list`` for Int, ``float_list`` for Float and ``bool_list`` for Boolean.
```

Comment:
also refer to the python types using backticks, so:
```
You can use shortcuts for basic types, ``str_list`` for ``str`` ,
``int_list`` for ``int``, ``float_list`` for ``float`` and ``bool_list`` for ``bool``.
```
Or write them out if you don't want to refer to the python types:

```
You can use shortcuts for basic types, ``str_list`` for strings,
``int_list`` for integers, ``float_list`` for floats and ``bool_list`` for boolean values.
```